packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -343,9 +412,10 @@ try {
     const message = event.data;
     switch (message.type) {
       case 'didUpdateGraphData':
-        const graphData = augmentGraphInfo(message.payload);
+        graphData = augmentGraphInfo(message.payload);
+        Actions.refreshWorkspaceData(graphData);
         console.log('didUpdateGraphData', graphData);
-        Actions.refresh(graphData);
+        Actions.filterByType();
```

Comment:
Oh I started from the bottom on these, I just prefer filtering by default instead of showing every node in the graph on load. If that's not desirable then it's an easy fix.